data/json/tool_qualities.json

```diff
@@ -29,7 +29,10 @@
     },{
         "type" : "tool_quality",
         "id" : "SAW_W",
-        "name" : "wood sawing"
+        "name" : "wood sawing",
+        "usages": [
+            [ 2, [ "LUMBER" ] ]
+        ]
```

Comment:
Why not keep it on one line? Short, one element arrays look clearer as one line.